demo-shell/src/app/components/process-service/process-service.component.html

```diff
@@ -238,18 +247,12 @@
     </mat-tab>
     <mat-tab id="report-header" href="#report"
              label="{{'PS-TAB.REPORTS-TAB' | translate}}">
-        <div class="app-grid" fxLayout="row" fxLayout.lt-lg="column" fxLayoutAlign="stretch">
-            <div class="app-grid-item app-reports-menu" fxFlex.gt-md="300px">
+        <div class="app-grid">
+            <div class="app-grid-item app-reports-menu">
                 <span><h5>Report List</h5></span>
                 <mat-divider></mat-divider>
-                <!-- <adf-analytics-report-list
```

Comment:
What about this commented out code? Can it be removed?

Reply:
Well, we should not have commented code and we always can find it in github history in case when it is needed. 
Also if you check history you can see it is here from ages so seems like it is unused and nobody needed it :D 
<img width="1071" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/82d96231-62cb-4c7a-970d-73b195cb5a62">